src/containers/BooksList.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Book from '../components/Book';
+
+function BooksList({ books }) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>
+        {
+          books.map(book => (
+            <Book key={book.id} id={book.id} title={book.title} category={book.category} />
```

Comment:
- [ ] [REQUIRED] Even though this works, it is required that you pass a `book` object as a prop to each `Book` component. This will keep your code concise and easy to read and understand.